assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -131,6 +131,9 @@ const ExternalActivity: React.FC<any> = (props) => {
     // result of init has a state snapshot with latest (init state applied)
     writeCapiLog('INIT RESULT CAPI', initResult);
     const currentStateSnapshot = initResult.snapshot;
+    if (initResult.context.currentActivity) {
+      setActivityId(initResult.context.currentActivity);
```

Comment:
Only setting this here as we want the owner activity Id to be appended. So far test this with KIP open and free, small word preview etc